smrf/tests/test_threading.py

```diff
@@ -1,15 +1,13 @@
-from copy import deepcopy
-
-from inicheck.tools import cast_all_variables
-
 from smrf.framework.model_framework import SMRF, run_smrf
 from smrf.tests.smrf_test_case_lakes import SMRFTestCaseLakes
 
 
 class TestThreading(SMRFTestCaseLakes):
+    THREAD_VARIABLE_COUNT = 36
 
-    def get_variables(self, config):
-        s = SMRF(self.base_config)
+    @staticmethod
+    def get_variables(config):
+        s = SMRF(config)
```

Comment:
Think there was a bug here, where the passed in config was not used and the base without threading instead.

Reply:
I was attempting to run it not threaded first then threaded to see if any of the variables changed.

Reply:
Should we keep the test or nuke this class?

Reply:
I don't know, I'm on the fence. It's useful in the sense it makes us aware of what is in the queue. But it's a lot of extra code for that check.

Or can we move the `assert_thread_variables` here for a quick test?

Reply:
That's sound like a perfect place for it.